decidim-proposals/spec/events/decidim/proposals/publish_proposal_event_spec.rb

```diff
@@ -59,6 +59,17 @@ module Proposals
         end
       end
 
+      context "when the proposal is official" do
+        let(:resource) { create :proposal, :official, title: "A nice proposal" }
+
+        describe "notification_title" do
+          it "is generated correctly" do
+            expect(subject.notification_title)
+              .to include("The official proposal <a href=\"#{resource_path}\">#{resource_title}</a> was published.")
```

Comment:
You'll also need to change the expected message accordingly to the changes here.